zookeeper-server/src/main/java/org/apache/zookeeper/server/ContainerManager.java

```diff
@@ -169,7 +175,6 @@ protected long getMinIntervalMs() {
 
     // VisibleForTesting
     protected long getElapsed(DataNode node) {
-        return Time.currentWallTime() - node.stat.getMtime();
+        return Time.currentWallTime() - node.stat.getCtime();
```

Comment:
This is a breaking change and is incorrect. The concept of a TTL node is that it expires after non-use. If the TTL is continually updated it should not be deleted.